--- settings-view.old.coffee	2014-10-24 15:39:06.000000000 +0100
+++ settings-view.coffee	2014-10-24 15:39:14.000000000 +0100
@@ -25,7 +25,7 @@
             @subview 'filterEditor', new TextEditorView(mini: true, placeholderText: 'Filter packages')
         @ul class: 'panels-packages nav nav-pills nav-stacked', outlet: 'panelPackages'
         @div class: 'button-area', =>
-          @button class: 'btn btn-default icon icon-link-external', outlet: 'openDotAtom', 'Open ~/.atom'
+          @button class: 'btn btn-default icon icon-link-external', outlet: 'openDotAtom', 'Open ~/.sparkdev'
       @div class: 'panels padded', outlet: 'panels'

   initialize: ({@uri, activePanelName}={}) ->
