--- atom-application.coffee.orig	2014-11-17 19:07:38.000000000 +0100
+++ atom-application.coffee	2014-11-17 19:24:34.000000000 +0100
@@ -16,9 +16,9 @@
 
 socketPath =
   if process.platform is 'win32'
-    '\\\\.\\pipe\\atom-sock'
+    '\\\\.\\pipe\\sparkdev-sock'
   else
-    path.join(os.tmpdir(), 'atom.sock')
+    path.join(os.tmpdir(), 'sparkdev.sock')
 
 # The application's singleton class.
 #
@@ -154,8 +154,8 @@
       atomWindow ?= @focusedWindow()
       atomWindow?.browserWindow.inspectElement(x, y)
 
-    @on 'application:open-documentation', -> require('shell').openExternal('https://atom.io/docs/latest/?app')
-    @on 'application:open-terms-of-use', -> require('shell').openExternal('https://atom.io/terms')
+    @on 'application:open-documentation', -> require('shell').openExternal('http://docs.spark.io/dev/')
+    @on 'application:open-terms-of-use', -> require('shell').openExternal('https://www.spark.io/tos')
     @on 'application:install-update', -> @autoUpdateManager.install()
     @on 'application:check-for-update', => @autoUpdateManager.check()
 
