--- atom.coffee.orig	2015-06-09 13:12:20.000000000 +0100
+++ atom.coffee	2015-06-09 16:18:43.000000000 +0100
@@ -590,12 +590,6 @@
   startEditorWindow: ->
     {safeMode} = @getLoadSettings()
 
-    CommandInstaller = require './command-installer'
-    CommandInstaller.installAtomCommand false, (error) ->
-      console.warn error.message if error?
-    CommandInstaller.installApmCommand false, (error) ->
-      console.warn error.message if error?
-
     @loadConfig()
     @keymaps.loadBundledKeymaps()
     @themes.loadBaseStylesheets()
