--- auto-update-manager.coffee.orig	2015-04-28 14:17:49.000000000 +0100
+++ auto-update-manager.coffee	2015-04-28 14:55:12.000000000 +0100
@@ -20,10 +20,10 @@
     if process.platform is 'win32'
       # Squirrel for Windows can't handle query params
       # https://github.com/Squirrel/Squirrel.Windows/issues/132
-      @feedUrl = 'https://atom.io/api/updates'
+      @feedUrl = 'https://updates.particle.io'
     else
       @iconPath = path.resolve(__dirname, '..', '..', 'resources', 'atom.png')
-      @feedUrl = "https://atom.io/api/updates?version=#{@version}"
+      @feedUrl = "https://updates.particle.io?version=#{@version}"

     process.nextTick => @setupAutoUpdater()
