--- codesign-task.coffee.1	2014-11-04 16:45:54.000000000 +0100
+++ codesign-task.coffee	2014-11-04 16:45:18.000000000 +0100
@@ -26,7 +26,7 @@
     switch process.platform
       when 'darwin'
         cmd = 'codesign'
-        args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: GitHub', grunt.config.get('atom.shellAppDir')]
+        args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: Spark Labs, Inc.', grunt.config.get('atom.shellAppDir')]
         spawn {cmd, args}, (error) -> callback(error)
       when 'win32'
         spawn {cmd: 'taskkill', args: ['/F', '/IM', 'atom.exe']}, ->
