--- codesign-task.coffee.orig	2015-08-07 16:43:11.000000000 +0100
+++ codesign-task.coffee	2015-08-07 16:47:16.000000000 +0100
@@ -17,7 +17,7 @@
   grunt.registerTask 'codesign:installer', 'Codesign AtomSetup.exe', ->
     done = @async()
     cmd = process.env.JANKY_SIGNTOOL ? 'signtool'
-    atomSetupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'AtomSetup.exe')
+    atomSetupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'ParticleDevSetup.exe')
     spawn {cmd, args: [atomSetupExePath]}, (error) -> done(error)
 
   grunt.registerTask 'codesign:app', 'Codesign Atom.app', ->
@@ -27,7 +27,7 @@
       return done(error) if error?
 
       cmd = 'codesign'
-      args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: GitHub', grunt.config.get('atom.shellAppDir')]
+      args = ['--deep', '--force', '--verbose', '--sign', 'Developer ID Application: Spark Labs, Inc.', grunt.config.get('atom.shellAppDir')]
       spawn {cmd, args}, (error) -> done(error)
 
   unlockKeychain = (callback) ->
