--- codesign-task.coffee.old	2015-07-20 15:37:26.000000000 +0100
+++ codesign-task.coffee	2015-07-20 15:37:31.000000000 +0100
@@ -36,7 +36,7 @@
           spawn {cmd, args}, (error) ->
             return callback(error) if error?
 
-            setupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'AtomSetup.exe')
+            setupExePath = path.resolve(grunt.config.get('atom.buildDir'), 'installer', 'ParticleDevSetup.exe')
             if fs.isFileSync(setupExePath)
               args = [setupExePath]
               spawn {cmd, args}, (error) -> callback(error)
