--- darwin.old.cson	2014-10-24 14:56:39.000000000 +0100
+++ darwin.cson	2014-10-24 14:58:17.000000000 +0100
@@ -1,8 +1,8 @@
 'menu': [
   {
-    label: 'Atom'
+    label: 'Spark Dev'
     submenu: [
-      { label: 'About Atom', command: 'application:about' }
+      { label: 'About Spark Dev', command: 'application:about' }
       { label: 'View License', command: 'application:open-license' }
       { label: 'VERSION', enabled: false }
       { label: 'Restart and Install Update', command: 'application:install-update', visible: false}
@@ -20,7 +20,7 @@
       { type: 'separator' }
       { label: 'Services', submenu: [] }
       { type: 'separator' }
-      { label: 'Hide Atom', command: 'application:hide' }
+      { label: 'Hide Spark Dev', command: 'application:hide' }
       { label: 'Hide Others', command: 'application:hide-other-applications' }
       { label: 'Show All', command: 'application:unhide-all-applications' }
       { type: 'separator' }
@@ -160,7 +160,7 @@
         label: 'Developer'
         submenu: [
           { label: 'Open In Dev Mode...', command: 'application:open-dev' }
-          { label: 'Run Atom Specs', command: 'application:run-all-specs' }
+          { label: 'Run Spark Dev Specs', command: 'application:run-all-specs' }
           { label: 'Run Package Specs', command: 'window:run-package-specs' }
           { label: 'Toggle Developer Tools', command: 'window:toggle-dev-tools' }
         ]
