--- linux.cson.orig	2015-07-20 13:25:42.000000000 +0100
+++ linux.cson	2015-07-20 13:44:11.000000000 +0100
@@ -121,8 +121,6 @@
         label: 'Developer'
         submenu: [
           { label: 'Open In &Dev Mode…', command: 'application:open-dev' }
-          { label: 'Run &Atom Specs', command: 'application:run-all-specs' }
-          { label: 'Run Package &Specs', command: 'window:run-package-specs' }
           { label: 'Toggle Developer &Tools', command: 'window:toggle-dev-tools' }
         ]
       }
