--- linux.old.cson	2014-10-24 14:56:18.000000000 +0100
+++ linux.cson	2014-10-24 14:58:40.000000000 +0100
@@ -98,7 +98,7 @@
         label: 'Developer'
         submenu: [
           { label: 'Open In &Dev Mode...', command: 'application:open-dev' }
-          { label: 'Run &Atom Specs', command: 'application:run-all-specs' }
+          { label: 'Run P&article Dev Specs', command: 'application:run-all-specs' }
           { label: 'Run Package &Specs', command: 'window:run-package-specs' }
           { label: 'Toggle Developer &Tools', command: 'window:toggle-dev-tools' }
         ]
