--- publish-build-task.coffee.old.coffee	2015-07-20 12:30:11.000000000 +0100
+++ publish-build-task.coffee	2015-07-20 12:29:52.000000000 +0100
@@ -121,10 +121,10 @@
   async.parallel(tasks, callback)
 
 getAtomDraftRelease = (callback) ->
-  atomRepo = new GitHub({repo: 'atom/atom', token})
+  atomRepo = new GitHub({repo: 'spark/spark-dev', token})
   atomRepo.getReleases (error, releases=[]) ->
     if error?
-      logError('Fetching atom/atom releases failed', error, releases)
+      logError('Fetching spark/spark-dev releases failed', error, releases)
       callback(error)
     else
       [firstDraft] = releases.filter ({draft}) -> draft
@@ -147,7 +147,7 @@
 createAtomDraftRelease = (callback) ->
   {version} = require('../../package.json')
   options =
-    uri: 'https://api.github.com/repos/atom/atom/releases'
+    uri: 'https://api.github.com/repos/spark/spark-dev/releases'
     method: 'POST'
     headers: defaultHeaders
     json:
@@ -162,7 +162,7 @@
 
   request options, (error, response, body='') ->
     if error? or response.statusCode isnt 201
-      logError("Creating atom/atom draft release failed", error, body)
+      logError("Creating spark/spark-dev draft release failed", error, body)
       callback(error ? new Error(response.statusCode))
     else
       callback(null, body)
