--- set-version.orig	2015-07-21 17:11:37.000000000 +0100
+++ set-version	2015-07-21 17:11:55.000000000 +0100
@@ -4,8 +4,8 @@
 
 BUILT_PRODUCTS_DIR=$1
 VERSION=$2
-PLIST_PATH="$BUILT_PRODUCTS_DIR/Atom.app/Contents/Info.plist"
-HELPER_PLIST_PATH="$BUILT_PRODUCTS_DIR/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/Info.plist"
+PLIST_PATH="$BUILT_PRODUCTS_DIR/Particle Dev.app/Contents/Info.plist"
+HELPER_PLIST_PATH="$BUILT_PRODUCTS_DIR/Particle Dev.app/Contents/Frameworks/Atom Helper.app/Contents/Info.plist"
 
 # Update version
 /usr/libexec/PlistBuddy -c "Set CFBundleShortVersionString $VERSION" "$PLIST_PATH"
