--- workspace.coffee.orig	2015-04-28 16:48:51.000000000 +0100
+++ workspace.coffee	2015-04-28 16:50:45.000000000 +0100
@@ -148,7 +148,7 @@
   # Updates the application's title and proxy icon based on whichever file is
   # open.
   updateWindowTitle: =>
-    appName = 'Atom'
+    appName = 'Particle Dev'
     projectPaths = atom.project?.getPaths() ? []
     if item = @getActivePaneItem()
       itemPath = item.getPath?()
